### DISCUSSION TITLE
Is there any way to re-check images which were marked as "not found"?

### DISCUSSION THREAD

**Discussion by Oleksandr-thescimus** [photoprism/photoprism#3880] [Q&A]
I faced a problem, while the first indexing my usb device was disconnected (there were originals on the device), and many files were marked as "not found".

Should I run "Complete rescan"? 
Or is there any way to re-check only images which were marked as "not found"?

**Comment by graciousgrey** [comment]
If the files have not yet been indexed, normal indexing should find them. I'm not 100% sure about files that were previously indexed but then removed from the index because the files were not accessible during indexing. I would recommend trying a normal index in a subfolder of originals first to see if that is enough.
